database/buildSchema/36_application_list_view.sql

```diff
@@ -0,0 +1,60 @@
+-- Aggregated VIEW table of all data required for application list page
+
+CREATE OR REPLACE VIEW "public".application_list AS
+ SELECT 
+	app.id, app.serial, app.name,
+	template.code as template_code,
+    template.name as template_name,
+	username as applicant_username,
+	first_name as applicant_first_name,
+	last_name as applicant_last_name,
+	CONCAT(first_name, ' ', last_name) as applicant,
+	org.name as org_name,
+	stage_status.stage, stage_status.status, outcome,
+	status_history_time_created as last_active_date,
+	(
+	WITH count_unassigned_sections AS (
```

Comment:
I would extract this into a method or view. (an not sure if the 'WITH' statement is needed, to me the best use of 'WITH' is for recursiveness/loops within a query).

Personally I would do it like this:

```PG

/* make into method 'application_reviewable_questions' */

SELECT count(*) 
FROM template_section
JOIN template_element on template_element.section_id = template_section.id
WHERE
	template_section.template_id = 4 /* from method parameter */ and template_element.category != 'Information'

/* make into method 'application_assigned_questions' */

SELECT count(distinct(template_element.id))
FROM review_assignment
JOIN review_question_assignment on review_question_assignment.review_assignment_id = review_assignment.id
JOIN template_element on template_element.id = review_question_assignment.template_element_id 
WHERE
	review_assignment.status = 'Assigned'
	AND review_assignment.stage_id = 5 /* from parameter */
	AND review_assignment.level = 1 /* from parameter */
	AND review_assignment.application_id = 4000 /* from parameter (application.id) */
	AND template_element.category != 'Information'

/* in application_lise view */

application_assigned_questions(app) >= application_reviewable_questions(app, state_id, 1) as is_fully_assigned
```

Here are two branches which i used to check results of the select statement:

https://github.com/openmsupply/application-manager-server/compare/%23196-is_fully_assigned-to-application-list...%23196-review-assignments-not-fully-assigned?expand=1

and 

https://github.com/openmsupply/application-manager-server/compare/%23196-is_fully_assigned-to-application-list...%23196-review-assignment-fully-assigned?expand=1